src/tests/svg.rs

```diff
@@ -0,0 +1,25 @@
+#[cfg(feature = "svg")]
+use crate::convert::svg::SvgBuilder;
+use crate::convert::Builder;
+use crate::{QRBuilder, ECL};
+use base64::{engine::general_purpose, Engine as _};
+
+#[cfg(feature = "svg")]
+#[test]
+fn it_embeds_an_image_via_data_uri() {
+    let image_bytes = std::fs::read("assets/red_square.png").unwrap();
+    let image_base64 = general_purpose::STANDARD.encode(&image_bytes);
+    let data_uri = format!("data:image/png;base64,{image_base64}");
```

Comment:
This data URI embeds the image directly into the SVG, not just as a link. Let me know if you'd like me to add an example for this as well.

Reply:
I think it's good too, do you think we should add the file asset & the dev dependency to test that?
Maybe a simple url / inline base64 could do the trick? :)

Reply:
Sure, I'll modify this to use a test base64 string and we can drop the extra dependency